pkg/artifactory/provider.go

```diff
@@ -2,6 +2,9 @@ package artifactory
 
 import (
 	"fmt"
+	"net/http"
```

Comment:
You just reordered the imports?

Reply:
vscode did it, let me amend it :)

Reply:
I applied `gofmt` to have consistent formatting.